frontend/packages/dev-console/src/components/monitoring/alerts/monitoring-alerts-utils.tsx

```diff
@@ -157,13 +152,15 @@ const setOrderBy = (orderBy: SortByDirection, data: Rule[]): Rule[] => {
   return orderBy === SortByDirection.asc ? data : data.reverse();
 };
 
-const alertingRuleNotificationsOrder = (rule: Rule) => [
-  rule.state === RuleStates.Silenced ? 1 : 0,
-  rule.state,
-];
+export const alertingRuleNotificationsOrder = (rule: Rule) => {
```

Comment:
verified locally. silencing works as expected.
just one nit: could you please add the return type for the function
`export const alertingRuleNotificationsOrder = (rule: Rule): number[] => {`